tests/integration/run.bats

```diff
@@ -154,8 +154,8 @@ function teardown() {
 	fi
 }
 
-@test "RUNC_DMZ=legacy runc run [/proc/self/exe clone]" {
-	RUNC_DMZ=legacy runc --debug run test_hello
+@test "[RUNC_DMZ=false] runc run [/proc/self/exe clone]" {
```

Comment:
ditto

Reply:
idem. And next time don'y worry, I can `git grep` :)